src/app/(main)/search/page.tsx

```diff
@@ -0,0 +1,17 @@
+import { SearchForm } from "@/app/(main)/search/search-form";
+import { SearchResult } from "@/app/(main)/search/search-results";
+import { usersService } from "@/services/users-service";
+import { auth } from "@clerk/nextjs";
+import { Suspense } from 'react'
+
+export default async function SearchPage() {
```

Comment:
https://nextjs.org/docs/app/api-reference/file-conventions/page#searchparams-optional

The `SearchPage` component can take in `searchParams` which is an object dictionary

inside the `SearchPage`, you can do something like

```tsx
const latitude = int(searchParams.latitude)
const longitude = int(searchParams.longitude)
...
```

Use this to fetch the host listings (calling the search service) then pass them into `SearchResults`

Reply:
If this is done, SearchResults will not longer have to have "use client"

Reply:
@arjbhat